GetFed/GetFed/Controllers/FoodDetailViewController.swift

```diff
@@ -127,9 +127,7 @@ extension FoodDetailViewController {
     
     func formatChartValues(with dataSet: PieChartDataSet?) {
         if let dataSet = dataSet {
-            let measurementFormatter = MeasurementFormatter()
-            let customValueFormatter = CustomValueFormatter(measurementFormatter: measurementFormatter)
-            dataSet.valueFormatter = customValueFormatter
+            dataSet.valueFormatter = CustomValueFormatter()
```

Comment:
If you're just mutating, you can just set it directly, regardless of whether a value is there: `dataSet?.valueFormatter = CustomValueFormatter()`.